packages/jest-jasmine2/src/queueRunner.js

```diff
@@ -29,13 +28,12 @@ type QueueableFn = {
 function queueRunner(options: Options) {
   const mapper = ({fn, timeout}) => {
     const promise = new Promise(resolve => {
-      const next = once(resolve);
-      next.fail = function() {
+      resolve.fail = function() {
```

Comment:
uh.. can you please not monkey-patch the `resolve` function? How about wrapping it?